BuildFile.xml

```diff
@@ -0,0 +1,15 @@
+<use name="FWCore/Utilities"/>
+<use name="FWCore/PluginManager"/>
+<use name="FWCore/Framework"/>
+<use name="root"/>
+<use name="DataFormats/PatCandidates"/>
+<use name="DataFormats/HepMCCandidate"/>
+<use name="RecoEgamma/EgammaTools"/>
+<use name="CommonTools/Utils" />
+<use name="cp3_llbb/TreeWrapper"/>
+<use name="cp3_llbb/Framework"/>
+<lib name="MathMore"/>
+<use name="rootrflx"/>
```

Comment:
For TT we had to remove this line for the dictionary generation to work properly...